veil-macros/src/lib.rs

```diff
@@ -1,87 +1,31 @@
+//! Macros for `veil`
+
+#![deny(missing_docs)]
```

Comment:
This could be a warning for development, the CI won't pass if the warnings are present due to `cargo clippy -- -D warnings`

Reply:
Good point, will do this in a separate PR too. PLATFORM-868

Reply:
I think you could put it in this one, since it doesn't require any changes to .drone.yml

Reply:
Ya, however there is a `#[deny(missing_docs)]` in the `veil` crate as well which is untouched/not added by this PR, so let's keep it consistent for now.

Reply:
:+1: